files/repair.js

```diff
@@ -0,0 +1,23 @@
+function repair (ship) {
+
+    const damagedHull = ship.subsystems.hull
+    const damagedShield = ship.subsystems.shield
+    const crew = ship.crew
+    const totalHull = ship.subsystems.totalHull
+    const totalShield = ship.subsystems.totalShield
```

Comment:
Die Variablen "ship.subsystems.totalShield" und "ship.subsystems.totalHull" gibt es nicht. Allerdings habe ich extra Konstanten angelegt, die speichern, wie der Start-Wert für die verschiedenen Subsysteme ist. Also auch, bis zu welchem Wert repariert werden kann: Z.B. playerMaxHull und playerMaxShield in der Datei PlayerShip.js. (Diese sind nicht im Schiff selbst, aber in der Datei.) Ich denke das ist, was du meinst?

Reply:
ja, solange beide geht, finde ich die gut. Wir können lassen wir du geschrieben hast, wobei 'maxX' deutlich beschreibt, was das macht. Es ist alles gut.

Reply:
könntest du aber, die Konflikte in Ordnung zu bringen? Mit git fühle ich mich noch nicht sicher genug und will nichts schieffes machen. Danke!!